tests/unit/test_openapi/test_parameters.py

```diff
@@ -324,3 +325,19 @@ async def index(
         assert response.json()["paths"]["/"]["get"]["parameters"][0]["examples"] == {
             "text-example-1": {"summary": "example summary", "value": "example value"}
         }
+
+
+def test_uuid_path_description_generation() -> None:
```

Comment:
I am also not sure if this is where the test should go 🙃. The test function above what I wrote seems to test something relevant so I added it here